### ISSUE TITLE
Explore Safari support

### ISSUE THREAD

**Issue by jorgegonzalez** [thebaer/MMRA#31]
I suspect that MMRA will work just fine in Safari, it just needs to be bundled as a Safari extension.

**Comment by thebaer** [comment]
Definitely. I've added the basic configuration on the `safari` branch and bundled everything as-is. If anyone with a little more experience wants to try it out and make tweaks, please feel free. The styles in `medium.css` load fine, but:

* I'm not sure of the best place to load `content.js` or
* how to let people access the Options page

**Comment by thebaer** [comment]
This is [available for Safari](https://safari-extensions.apple.com/details/?id=com.abunchtell.mmra-TPPAB4YBA6) now! But it pretty much only has the CSS rules, so no getting rid of sticky bars or large images, and no options.

We could use some help from anyone more experienced with building Safari extensions for getting the rest of this extension's functionality in. You can find everything in the [MMRA.safariextension](https://github.com/thebaer/MMRA/tree/master/MMRA.safariextension) folder (CSS and JS has been copied into it).
